src/Components/test/E2ETest/Microsoft.AspNetCore.Components.E2ETests.csproj

```diff
@@ -47,10 +47,11 @@
     <ProjectReference Include="..\testassets\TestServer\Components.TestServer.csproj" />
     <ProjectReference Include="..\..\WebAssembly\testassets\Wasm.Authentication.Server\Wasm.Authentication.Server.csproj" />
 
+    <!-- TODO - turn this back on once the SDK is higher than 6.0.100-preview.5.21230.2: https://github.com/dotnet/aspnetcore/issues/32788
     <ProjectReference Include="..\..\benchmarkapps\Wasm.Performance\TestApp\Wasm.Performance.TestApp.csproj"
```

Comment:
You'll also have to skip the tests

Reply:
Which tests?

Reply:
According to https://dev.azure.com/dnceng/public/_build/results?buildId=1142465&view=ms.vss-test-web.build-test-results-tab,
- `BenchmarksRunWithoutError`
- `HasTitle`